optimum/commands/export/openvino.py

```diff
@@ -329,11 +329,18 @@ def run(self):
             model.save_pretrained(self.args.output)
             if not self.args.disable_convert_tokenizer:
                 maybe_convert_tokenizers(library_name, self.args.output, model, task=task)
-        elif task.startswith("text-generation") and quantize_with_dataset:
-            from optimum.intel import OVModelForCausalLM
+        elif (task.startswith("text-generation") or task == "image-text-to-text") and quantize_with_dataset:
+            if task.startswith("text-generation"):
```

Comment:
not an issue, just an idea, we may have some mapping between tesk and model inference class in future to simplify initialization process. Additionally we can provide message with recommendation which class to use orr create OVAutoModel class that will load model to specific task based on exported model config (using task and info about model type if necessary)